pkg/bookings/bookings.go

```diff
@@ -0,0 +1,21 @@
+package bookings
+
+import (
+	"context"
+
+	"github.com/clearchanneloutdoor/io-sdk-golang/pkg/clients"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+const (
+	serverUrl string = "https://direct.cco.io"
```

Comment:
This is good, we'll modify the pxy-api clients.yaml for each environment to include the order-api and booking-api... for your immediate needs, however, when using the SDK, you'll want to use the variadic field on NewClient (line 14 below) to provide the Istio address of the API you're hitting:

```golang
bc, err := bookings.NewClient(ctx.Background(), oaCfg, "booking-api.booking-api.svc")
if err != nil {
  // argh...
}
```